.travis.yml

```diff
@@ -27,4 +27,19 @@ env:
 
 install: beaver dlang install
 
-script: beaver dlang make
+# Build packages too, this should be done by beaver:
+# https://github.com/sociomantic-tsunami/beaver/issues/34
+script: beaver dlang make && beaver dlang make pkg
+
+jobs:
+    include:
+        - stage: Upload Package
+          # We need to include the exclusion of D2 tags because this "if"
+          # replaces the global one
+          if: tag IS present
+          # Which package to deploy
+          env: DMD=dmd1 DIST=xenial F=production
```

Comment:
Ok so this currently just deploys the D1 build, right? (Fine for now. I guess we can make it deploy both when we want to try D2 again.)

Reply:
It would be good to clarify if we can get `dhtnode-d2` package upload for free, so we don't need to juggle devinfra when we start deploying d2.

Reply:
Yeah, we could upload both I guess. I would need to play a bit to see if we can just use a matrix in this stage or if we should just add a new stage for it.

Reply:
Maybe we can merge this as is, try it out, and if it works, then go for the D2 package too.

Reply:
I'll allow someone with more dev-infra knowledge than I possess to do the merge.